dhis-2/dhis-web/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramController.java

```diff
@@ -111,4 +118,27 @@
 
         return MetadataExportControllerUtils.getWithDependencies( contextService, exportService, program, download );
     }
+
+    @ResponseBody
+    @RequestMapping( value = "orgUnits" )
+    IdentifiableObjectAssociations getProgramOrgUnitsAssociations(
+        @RequestParam( value = "programs" ) Set<String> programUids, User currentUser )
+    {
+
+        if ( Objects.isNull( programUids ) || programUids.size() == 0 )
+        {
+            throw new IllegalArgumentException( "At least one program uid must be specified" );
+        }
+
+        if ( !aclService.canRead( currentUser, getEntityClass() )
```

Comment:
I dont think this is necessary here. We never explicitly set any authorities for reading Program and OrganisationUnit. But lets wait and see what others have to say.

Reply:
fixed, thank you